payments/swish/redirect.md

```diff
@@ -162,10 +161,12 @@ Content-Type: application/json
         },
         "prefillInfo": {
             "msisdn": "+46739000001"
-        },
-        "swish": {
-            "ecomOnlyEnabled": false
         }
+    },
+    "swish": {
```

Comment:
Since the variables has been moved into a new object this new object has to be described and added to this table.